src/cartridges/int_adyen_SFRA/cartridge/client/default/js/expressPaymentMethodsVisibility.js

```diff
@@ -0,0 +1,20 @@
+async function handleExpressPaymentsVisibility() {
+  const { expressMethodsOrder } = window;
+  if (expressMethodsOrder) {
+    const sortOrder = expressMethodsOrder.split(',');
+    const container = document.getElementById('express-container');
+    let toSort = container.children;
+    toSort = Array.prototype.slice.call(toSort, 0);
```

Comment:
could we use "container.children" as "call" function parameter then we would not need line 6 assignment.

Reply:
great suggestion 👍 done!